server/cyf.sql

```diff
@@ -0,0 +1,20 @@
+DROP TABLE IF EXISTS energizers CASCADE;
+DROP TABLE IF EXISTS raitings CASCADE;
+
+CREATE TABLE energizers (
+    id           SERIAL PRIMARY KEY,
+    name         VARCHAR(50) NOT NULL,
```

Comment:
Is there any reason this is limited to 50 characters?

Reply:
The name should be short and descriptive